.circleci/config.yml

```diff
@@ -0,0 +1,40 @@
+# Use the latest 2.1 version of CircleCI pipeline process engine.
+# See: https://circleci.com/docs/2.0/configuration-reference
+version: 2.1
+
+commands:
+  command-build_and_test:
+    steps:
+      - run:
+          name: UIKit for React build and test
+          command: |
+            npm install
+            npm run build
+            npm test
```

Comment:
This is `yarn` tho :(

Reply:
https://github.com/sendbird/sendbird-uikit-react/blob/main/.github/workflows/test-n-lint.yml#L24